supervisor/const.py

```diff
@@ -270,7 +271,11 @@
 ATTR_SERVERS = "servers"
 ATTR_SERVICE = "service"
 ATTR_SERVICES = "services"
+ATTR_REMOTE_USER = "remote_user"
+ATTR_REMOTE_USER_REQUESTED = "remote_user_requested"
 ATTR_SESSION = "session"
+ATTR_SESSION_DATA = "session_data"
+ATTR_SESSION_DATA_USER_ID = "user-id"
```

Comment:
```suggestion
ATTR_SESSION_DATA_USER_ID = "user_id"
```

same as in core, we use `_` for json body data